kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UserTaskServicesClientImpl.java

```diff
@@ -870,6 +870,37 @@ public TaskInstance findTaskById(Long taskId) {
     }
 
     @Override
+    public List<TaskSummary> findTasksAssignedAsPotentialOwner(String userId, String filter, List<String> status, Integer page, Integer pageSize) {
```

Comment:
I meant to reuse findTasksAssignedAsPotentialOwner(String userId, String filter, List<String> status, Integer page, Integer pageSize, String sort, boolean sortOrder) - the other method added by this PR . In this method there is sent same JMS command, just with default sorting values.